x-pack/plugins/ml/server/models/job_service/jobs.ts

```diff
@@ -319,6 +320,12 @@ export function jobsProvider(
     if (jobResults && jobResults.jobs) {
       const job = jobResults.jobs.find((j) => j.job_id === jobId);
       if (job) {
+        // 'Managed' means job is preconfigured and deployed by other solutions
+        // we should not clone the setting
+        if (job.custom_settings?.managed === true) {
+          job.custom_settings.managed = false;
```

Comment:
We should just remove `managed` rather than setting it to `false`.
I have a draft PR here which is very similar. I included `managed` as this code is closely related.